cligpt.zsh

```diff
@@ -38,7 +38,13 @@ cligpt() {
     echo "Output: $COMMAND"
   elif [ -n "$COMMAND" ]; then
     echo "-> $COMMAND"
-    read -q "REPLY?Do you want to run this command? (y/n)"
+    if [ -n "$BASH_VERSION" ]; then
+      read -r -p "Do you want to run this command? (y/n) "
+    elif [ -n "$ZSH_VERSION" ]; then
+      read -r -q "Do you want to run this command? (y/n) "
```

Comment:
In the ZSH path, revert this read function back to the original.

Also use the same `current_shell` approach suggested in `install.sh`